### DISCUSSION TITLE
saf-training/courses/user/05

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#249] [Ideas]
# saf-training/courses/user/05

5. From "Plan" to "Validate" After identifying software components for your environment and knowing what security guidance exists for those components, a great next step is vali...

https://mitre.github.io/saf-training/courses/user/05.html

**Comment by karikarshivani** [comment]
Heimdall supported formats list may need to be updated

**Reply by p-oneil** [reply]
I don't see this list on this page

**Comment by karikarshivani** [comment]
Change the profile link in section 5.3.1 to a MITRE repo instead of CMSgov to standardize examples

**Comment by p-oneil** [comment]
Moved to PR
